frontend/ui/detailview.jsx

```diff
@@ -420,6 +428,25 @@ class DetailViewComponent extends React.Component {
   }
 
   render() {
+    let crashStatsLink;
+    if (this.state.measure in CRASH_STATS_MAPPING) {
+      const { processType, extraParams } = CRASH_STATS_MAPPING[this.state.measure];
+      const queryParams = stringify({
+        ...Object.keys(extraParams).reduce((dict, key) => ({ ...dict, [key]: extraParams[key] }), {}),
+        product: 'Firefox',
+        version: _.uniq(_.map(this.props.measureData, data => data.version)),
```

Comment:
We should exclude versions here that users have explicited turned off. Look for `disabledVersions` in this file.